src/styles/main.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables.scss";
+@import "./blocks/header.scss";
+@import "./blocks/catalog.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/card.scss";
+@import "./utils/reset.scss";
```

Comment:
You need to do reset import first of all, and it`s not actually reset, it`s looks normalize